sdk/maps/AzureMapsGeolocation/Source/GeolocationClient.swift

```diff
@@ -34,20 +34,21 @@ public class GeolocationClient {
     public init(
         endpoint: URL? = nil,
         credential: TokenCredential,
-        withOptions options: GeolocationClientOptions
-    ) throws {
-        self.service = try GeolocationClientInternal(
-            url: nil,
+        options: GeolocationClientOptions = GeolocationClientOptions()
+    ) {
+        service = try! GeolocationClientInternal(
```

Comment:
I don't think it's good idea to hide force try from end user.

Reply:
I dit it because this `try` is unnecessary, cause nothing is thrown from the internal client initialization